src/includes/configuration/sample-rate/dotnet.mdx

```diff
@@ -1,6 +1,7 @@
 ```csharp
 using Sentry;
 
+// Add this to the SDK initialization callback
 using (SentrySdk.Init(o =>
```

Comment:
Still doing init

Reply:
oops

Reply:
here we have `// app code here` so i just left it as it is.
removed my comment.